src/onefetch/info.rs

```diff
@@ -256,7 +256,11 @@ impl std::fmt::Display for Info {
         let center_pad = "   ";
         let mut info_lines = buf.lines();
 
-        if let Some(custom_image) = &self.config.image {
+        if self.config.image_off {
+            while let Some(line) = info_lines.next() {
+                writeln!(f, "{}", line)?;
+            }
```

Comment:
I don't think you need to iterate over info_lines, `writeln!(f, "{}\n", buf)?` should be enough